build_iso/sles12.rb

```diff
@@ -18,9 +18,17 @@
 require "rubygems"
 require "fileutils"
 
+def iso_repo
```

Comment:
Yes, especially the main Rakefile is quite low level, and does not help you if you still find your way around all the unfamiliar tools and all the images and VMs that are being handled.